src/_data/content.json

```diff
@@ -138,77 +138,82 @@
     },
```

Comment:
Your version of master was outdated. Please got to [master/src/_data](https://github.com/technologiestiftung/citylab-hackthecrisis/blob/master/src/_data/content.json#L133) and replace the full `content.json` in your branch, do your update and commit it again. Solving that conflict in the GitHub UI is tedious.